test_howdoi.py

```diff
@@ -100,6 +105,13 @@ def test_all_text(self):
         self.assertNotEqual(first_answer, second_answer)
         self.assertNotEqual(re.match('.*Answer from http.?://.*', second_answer, re.DOTALL), None)
 
+    def test_json_output(self):
+        query = self.queries[0]
```

Comment:
Inconsistent with last comment. We should either go for all `random` or all `query[0]`, although I am of the opinion that we should not use `random.choice` to test.

Reply:
Let's use the first `[0]`